### ISSUE TITLE
No error.. 

### ISSUE THREAD

**Issue by rathaROG** [gineshidalgo99/caffeCompilerForWindowsAndCUDA#1]
Hi, I just want to thank you! I successfully built on Windows 11, cuda 11.3.1, cudnn 8.3.3.40, custom build opencv 4.5.5, and latest boost 1.78. It was such an adventure. I will build openpose soon and see if it works well on the lastest RTX 3000 series :D

![Screenshot (2)](https://user-images.githubusercontent.com/10178493/160215061-a4e270f5-0f38-4a9f-b950-9605a0e10bb4.png)

**Comment by rathaROG** [comment]
Just update parts for openpose --> it doesn't run :'(

![image](https://user-images.githubusercontent.com/10178493/160256849-334c65ab-98c2-4311-b559-fe3e5eb43c8b.png)

**Comment by gineshidalgo99** [comment]
Yes, I have not been able to make the 3000 series work with Caffe yet (which is even harder for me because I do not have a 3000 card to even test it on it), but happy of any PR if you get any working workaround!!
